modules/runes/api/httphandler/get_utxos_output_by_location.go

```diff
@@ -0,0 +1,92 @@
+package httphandler
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/cockroachdb/errors"
+	"github.com/gaze-network/indexer-network/common/errs"
+	"github.com/gaze-network/indexer-network/modules/runes/runes"
+	"github.com/gaze-network/indexer-network/modules/runes/usecase"
+	"github.com/gofiber/fiber/v2"
+	"github.com/samber/lo"
+)
+
+type getUTXOsOutputByTxIdRequest struct {
```

Comment:
make request type match the handler
```suggestion
type getUTXOsOutputByLocationRequest struct {
```